Jenkinsfile

```diff
@@ -1 +1,8 @@
-python {}
+python {
+  owner = 'tools'
+  slackChannel = 'tools-eng'
+  // uses docker for testing, but nor for publishing.
+  nodeLabel = 'docker-oraclejdk8-compose'
+  usesDockerForTesting = true
+  dockerPush = false
```

Comment:
Probably we don't need `dockerPush` cause it is only defined in `dockerfile.groovy` ? And `owner` and `slackChannel` are all same as default config.

Reply:
You are right. I am being explicit in case we refactor `jenkins-common` later and foot-gun.